### ISSUE TITLE
too many prisoners spawned ?

### ISSUE THREAD

**Issue by x3003** [Realm667/WolfenDoom#584]
In c3m1_a, are there too many prisoners spawned and why ?!
When I look at vids from yt-ers who play c3m1, there are few prisoners in the beginning.
When I'm playing there are so many.

This pic is from a yt vid (but other vids are similar).

![boa_prisoners](https://user-images.githubusercontent.com/77861029/117455671-b5a59a80-af47-11eb-9ca5-14271c680d87.jpg)

This is from a freshly installed game.

![Screenshot_BoA_20210507_151234](https://user-images.githubusercontent.com/77861029/117454991-ffda4c00-af46-11eb-846d-0af0afde3fe6.png)

**Comment by AFADoomer** [comment]
You skipped the intro cinematic.  They didn't. The prisoners in their game are already ahead in the camp area.

**Comment by Ozymandias81** [comment]
Exactly, they walk towards the courtyard since you are arrived by train there a while ago.

**Comment by x3003** [comment]
Ahhh, another hidden secret :)

**Comment by x3003** [comment]
Well, i've played it so many times, that i'm not gonna wait till the cutscene is over ;)
